### DISCUSSION TITLE
playstore addition

### DISCUSSION THREAD

**Discussion by c0fe** [bromite/bromite#1321] [Ideas]
This is a great app but I am surprised that it isn't included in the Google Playstore, is there a particular reason for this?

**Comment by anuraganand-15** [comment]
Take a look at this :
https://github.com/bromite/bromite/blob/master/FAQ.md#is-bromite-on-play-store

**Comment by chromer030** [comment]
Is Bromite on Play Store?
No, and this is unlikely to change. Many limitations apply for submissions there, including which ads are allowed to be blocked.

Bromite favors user freedom in software choice: the device is yours so you get to choose which software to run on it, end of the story.

**Comment by spiderman973** [comment]
If you don't want to download bromite from f-droid you can install it from github/bromite official website
